R/edit.R

```diff
@@ -167,3 +168,11 @@ git_ignore_path <- function(scope = c("user", "project")) {
 
   path
 }
+
+#' @export
+#' @rdname edit
+edit_workflow <- function() {
```

Comment:
edit_workflow can be simpler than above functions (think @VerenaHeld and myself) because `.github/main.workflow` *only* exist at repo, never user level (as opposed to a `.gitignore`).

Reply:
Seems right.